### DISCUSSION TITLE
how to get attachment's content

### DISCUSSION THREAD

**Discussion by HQ-Panda** [cloudbees/zendesk-java-client#465] [Q&A] [answered]
i get attaachment by this 
comment.getAttachments();
get the attachment object,how can i get the contents of the file from this object

**Comment by aheritier** [comment, marked as answer]
Hi @xg578962913 

Use an HTTP client (You can use async-http-client like our API or any other) to download it from `attachment.getContentUrl()` ( don't forget to send your credentials ).

Best regards
